web/docs/about-vast/vision.md

```diff
@@ -33,21 +33,34 @@ moves away from point-to-point integrations:
 ![Security Data Fabric](/img/security-data-fabric.light.png#gh-light-mode-only)
 ![Security Data Fabric](/img/security-data-fabric.dark.png#gh-dark-mode-only)
 
+The network of VAST nodes forms the security data fabric. Communication takes
+place over a pluggable messaging *backbone*, such as Kafka, RabbitMQ, or MQTT.
+In this architecture, VAST assumes a mediator function, with a backbone-facing
+and local-facing side. On the backbone-facing side, VAST implements the security
+domain knowledge and analytical processing power to enable composable use cases,
+such as routing security content to detection engines, or executing security on
+top of the available telemetry. On the local-facing side, VAST integrates
+security tools by infusing them with fabric-provided data and, vice versa, and
```

Comment:
"and, vice versa, and" doesn't read smooth IMHO. A single "and" might be all we need here.